src/components/addCaption/AddCaption.tsx

```diff
@@ -102,20 +102,14 @@ const AddCaption = ({onClickSubmit, selectedImage}: any) => {
             </div>
           </div>
           <div className="flex justify-between mt-8 space-x-3 !justify-end">
-            <NavLink
-              to="/"
-              // className="flex items-center font-Bold leading-none max-w-full transition duration-100 outline-none bg-sky-300 focus:outline-none justify-center bg-icoBlue text-white hover:bg-sky-400 focus:bg-icoBlue-300 focus:ring-2 focus:ring-icoBlue-200 focus:ring-offset-2 focus:text-white px-10 py-5 min-h-14 min-w-150 text-sm rounded-md disabled:bg-icoGray-300 dark:disabled:bg-white/5 disabled:text-icoGray-400 dark:disabled:text-icoDarkMode-wolf disabled:cursor-not-allowed"
-            >
-              ◀ Previous: Calendar
-            </NavLink>
-            <button
-              // className="flex items-center font-Bold leading-none max-w-full transition duration-100 outline-none bg-sky-500 focus:outline-none justify-center bg-icoBlue text-white hover:bg-sky-400 focus:bg-icoBlue-300 focus:ring-2 focus:ring-icoBlue-200 focus:ring-offset-2 focus:text-white px-10 py-5 min-h-14 min-w-150 text-sm rounded-md disabled:bg-icoGray-300 dark:disabled:bg-white/5 disabled:text-icoGray-400 dark:disabled:text-icoDarkMode-wolf disabled:cursor-not-allowed"
+            <NavButton buttonText="<-return" to="/"></NavButton>
```

Comment:
Agree with this change and thanks for adding reusable component. I will change buttons to reusable once in my part, too